nanoFramework.System.Net.Http/Http/HttpContent.cs

```diff
@@ -94,7 +89,7 @@ public void CopyTo(Stream stream)
         /// </para>
         /// </remarks>
         /// <exception cref="ObjectDisposedException">If the object has been disposed.</exception>
-        public Stream LoadIntoBuffer()
+        public void LoadIntoBuffer()
```

Comment:
Any reason you decided to change the returned type?
I know the .NET LoadIntoBufferAsync does return a Task and we don't have task. So I'm not against having as void. But I'm bit afraid of having a breaking change here. If there is a great reason why having it as void, I would then have still the other function marked as deprecated.

Reply:
Oh right, I forgot that was a breaking change.
To be honest, the reason I did that change is to make the API resemble the .NET version more closely and because it doesn't really make sense to me for that method to return anything as the `ReadAsX`-Methods are the "correct" / intended way to read the data, no matter if it was loaded into the buffer before or not.

Which method would you like to mark as deprecated? We can't mark `LoadIntoBuffer` deprecated because it needs to be used (but returning void in my opinion) and we shouldn't mark `ReadAsStream` deprecated since that's the intended way to read the data as a stream.

Reply:
Not sure if that can be done because of the names and return types:

```csharp
public void LoadIntoBuffer()

[Deprecated]
public Stream LoadIntoBuffer()
{
  LoadIntoBuffer()
  return _buffer;
}
```

LoadIntBufferAsync is a valid .NET API and should be there. So if not possible to do something like in the pseudo code, then I would keep the one with stream.
Once we'll move to generics, Task will be possible and so, we should be able to remove the pure sync one.

Reply:
I'd love if it worked like that but it doesn't (methods with the same parameter signature can't have different return types).

Personally, I would rather prefer to introduce a breaking change that is simple to fix in code that uses the library (to be equivalent to .NET), rather than holding on to this behavior that is different to .NET forever.

But if you feel like that would be too much to ask of library users, I would propose returning a stream but adding a code comment that this isn't available in .NET and `ReadAsStream` should be used instead.

Reply:
Cleaner way is indeed to get rid of the stream and just have void. Agree on that. Now, we will know if that breaks on any of our own libs after merge with the automatic bump as all is rebuild. It's an easy fix anyway and much cleaner. 
So go for the breaking change then. @josesimoes any other opinion?

Reply:
Considering that we don't a proper way to go for the obsolete API and provide a new one... I'm good with this breaking change. 
Any code using it and expecting a stream will break anyway, therefore this won't go unnoticed, which would be the top concern.  
I'm labeling this as a breaking change too.